build.sbt

```diff
@@ -49,30 +51,30 @@ lazy val server = (project in file("server"))
       filters,
       jdbc,
       evolutions,
-      cache,
+      cacheApi,
       ws,
-      "org.scala-exercises" %% "exercises-stdlib"        % v('stdlib) xscalaExercises,
-      "org.scala-exercises" %% "exercises-cats"          % v('cats) xscalaExercises,
-      "org.scala-exercises" %% "exercises-shapeless"     % v('shapeless) xscalaExercises,
-      "org.scala-exercises" %% "exercises-doobie"        % v('doobie) xscalaExercises,
-      "org.scala-exercises" %% "exercises-scalacheck"    % v('scalacheck) xscalaExercises,
-      "org.scala-exercises" %% "exercises-fpinscala"     % v('fpinscala) xscalaExercises,
-      "org.scala-exercises" %% "exercises-scalatutorial" % v('scalatutorial) xscalaExercises,
-      "org.scala-exercises" %% "exercises-circe"         % v('circe) xscalaExercises,
-      "org.scala-exercises" %% "exercises-fetch"         % v('fetch) xscalaExercises,
-      "org.scala-exercises" %% "exercises-monocle"       % v('monocle) xscalaExercises,
-      "org.scala-exercises"     %% "evaluator-client" % v('evaluator) changing (),
-      "org.scala-exercises"     %% "runtime"          % version.value changing (),
-      "org.postgresql"          % "postgresql"        % v('postgres),
-      "com.vmunier"             %% "scalajs-scripts"  % v('scalajsscripts),
-      "com.lihaoyi"             %% "upickle"          % v('upickle),
-      "org.webjars"             %% "webjars-play"     % v('webjars),
-      "org.webjars"             % "bootstrap-sass"    % v('bootstrap),
-      "org.webjars"             % "highlightjs"       % v('highlightjs),
-      "com.tristanhunt"         %% "knockoff"         % v('knockoff),
-      "com.newrelic.agent.java" % "newrelic-agent"    % v('newrelic),
-      "commons-io"              % "commons-io"        % v('commonsio),
-      %%("freestyle"),
+      //"org.scala-exercises" %% "exercises-stdlib" % v('stdlib) xscalaExercises,
```

Comment:
Could we simply remove the unnecessary dependencies, even it they could come back in the future.